Backend/dev/migrations/dynamic-offer-driver-app/0157-add-colums-to-driverStats.sql

```diff
@@ -0,0 +1,21 @@
+-- Adding Columns in Driver Stats
+ALTER TABLE atlas_driver_offer_bpp.driver_stats add COLUMN total_rides int DEFAULT 0;
+ALTER TABLE atlas_driver_offer_bpp.driver_stats add COLUMN total_distance double precision DEFAULT 0;
+
+-- Updating total distance and total rides in ride_stats table
+UPDATE atlas_driver_offer_bpp.driver_stats
+    SET total_rides = ride_counts.count,
+        total_distance = ride_counts.distance_sum
+    FROM (
+        SELECT driver_id, COUNT(*) AS count, SUM(chargeable_distance) AS distance_sum
+        FROM atlas_driver_offer_bpp.ride
+        WHERE status = 'COMPLETED'
+        GROUP BY driver_id
+    ) AS ride_counts
+    WHERE ride_counts.DriverId = rideStats.DriverId;
+
+-- Add column in Transporter Configs
```

Comment:
Add `not null` constraint also to `total_rides` and `total_distance` field after updated value.